jobs/build/build-sync/Jenkinsfile

```diff
@@ -0,0 +1,149 @@
+
+
+properties(
+    [
+        buildDiscarder(
+            logRotator(
+                artifactDaysToKeepStr: '',
+                artifactNumToKeepStr: '',
+                daysToKeepStr: '',
+                numToKeepStr: '360'
+            )
+        ),
+        [
+            $class : 'ParametersDefinitionProperty',
+            parameterDefinitions: [
+                [
+                    name: 'MOCK',
+                    description: 'Mock run to pickup new Jenkins parameters?',
+                    $class: 'hudson.model.BooleanParameterDefinition',
+                    defaultValue: false,
+                ],
+                [
+                    name: 'BUILD_VERSION',
+                    description: 'OCP Version to build',
+                    $class: 'hudson.model.ChoiceParameterDefinition',
+                    choices: [
+			"4.0",
+			"4.1",
+		    ].join("\n"),
+                    defaultValue: '4.0'
+                ],
+                [
+                    name: 'TARGET_NODE',
+                    description: 'Jenkins agent node',
+                    $class: 'hudson.model.StringParameterDefinition',
```

Comment:
@sosiouxme didn't you figure out something WRT the target node? Doesn't we _not have to actually do that_?

Reply:
yeah, i don't know of any reason to set it at all.

```
node {
   .. do stuff...
}
```